MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Clock.xaml

```diff
@@ -547,4 +759,2919 @@
         </Style.Triggers>
     </Style>
 
+
+    <Style x:Key="MaterialDesignClockVertical" TargetType="{x:Type wpf:Clock}">
+        <Setter Property="Height" Value="auto"/>
+        <Setter Property="Width" Value="auto"/>
```

Comment:
Are these setters needed? Aren't these the default values for the properties?

Reply:
You're right, not needed. In the current implementation there are hardcoded dimensions, and I was experimenting with leaving it on auto by default.